packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -20,6 +20,11 @@ export abstract class DbtBaseProjectAdapter implements ProjectAdapter {
 
     catalog: DbtRpcDocsGenerateResults | undefined;
 
+    public async test(): Promise<void> {
+        await this.rpcClient.installDeps();
```

Comment:
The dependencies are only for generating new dbt models and don't affect the connection so you could remove them!